crates/flox/src/commands/mod.rs

```diff
@@ -218,20 +218,21 @@ enum Commands {
     Additional(#[bpaf(external(additional_commands))] AdditionalCommands),
     Internal(#[bpaf(external(internal_commands))] InternalCommands),
 }
+///flox is a virtual environment and package manager all in one. With flox you create development environments that layer and replace dependencies just where it matters, making them portable across the full software lifecycle.
 
 /// Local Development Commands
 #[derive(Bpaf, Clone)]
 enum LocalDevelopmentCommands {
     /// Create an environment in the current directory
     #[bpaf(command, long("create"))]
     Init(#[bpaf(external(environment::init))] environment::Init),
-    /// Activate environment
+    /// Enter the environment
     #[bpaf(command, long("develop"))]
     Activate(#[bpaf(external(environment::activate))] environment::Activate),
-    /// Search for packages to install
+    /// Search for system or library packages to install
```

Comment:
```suggestion
    /// Search for packages to install
```
```suggestion
    /// Search for executables or libraries to install
```
```suggestion
    /// Search for executable or library packages to install
```
Differentiating `system` and `library` packages is likely to get comments from pedants because most people will have experience installing libraries using their system package manager (e.g. `apt`, `brew`, etc). They (including me) would consider the libraries they're installing to be system packages because they're being installed system-wide as opposed to something like a Python library they're installing for a single project.

Reply:
we've tested the sentence on a few already. is this a blocking comment?

Reply:
Nope, just my input

Reply:
lets just ship as is. we can amend later but this one went through several turns already. :armadillo:?